tests/testthat/test-RunStep.R

```diff
@@ -45,3 +45,12 @@ test_that("Handles multiple parameters and function invocation correctly", {
   expect_equal(result$b, 300)
   expect_equal(result$c, "some_value")
 })
+
+test_that("RunStep will run a function from a namespace", {
+  lStep <- list(name = "dplyr::glimpse", params = list(head(Theoph)))
+  lData <- list(data1 = 300)
+  lMeta <- list(meta1 = 400)
+
+  result <- RunStep(lStep, lData, lMeta)
+  expect_equal(result, head(Theoph))
+})
```

Comment:
Add a test for a non-namespaced function to make sure it doesn't ever break anything.

Reply:
test added!